Octokit/Models/Response/Issue.cs

```diff
@@ -45,7 +45,7 @@ public Issue(string url, string htmlUrl, string commentsUrl, string eventsUrl, i
         /// <summary>
         /// The internal Id for this issue (not the issue number)
         /// </summary>
-        public int Id { get; private set; }
+        public long Id { get; private set; }
```

Comment:
`int id` in the constructor to be fixed as well

Reply:
Good catch - fixed in https://github.com/octokit/octokit.net/pull/2890/commits/b1d53cc49bdaf560a71431113babaecc355308cb and https://github.com/octokit/octokit.net/pull/2890/commits/cc2dd92b072310f32bff72bb69156f068259f546